tutorials/poromechanics.ipynb

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    class BoundaryConditionsMomentumBalance:
</pre><p>I suggest to not change the BCs of the momentum balance. The reason is that they are differentiated wrt time and should strictly speaking be TimeDependent (ad sense). I think the tutorial will not suffer from having the default conditions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>